intuita-webview/src/jobDiffView/DiffViewer/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { useRef, useState } from 'react';
+import ReactDiffViewer from 'react-diff-viewer-continued';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { useElementSize } from '../hooks/useElementSize';
+import { Container } from './Container';
+import { JobDiffViewProps } from '../App';
+
+enum JobKind {
+	rewriteFile = 1,
+	createFile = 2,
+	deleteFile = 3,
+	moveFile = 4,
+	moveAndRewriteFile = 5,
+	copyFile = 6,
+}
+
+export const JobDiffView = ({
+	jobKind,
+	oldFileContent,
+	newFileContent,
+	oldFileTitle,
+	newFileTitle,
+	title,
+}: JobDiffViewProps) => {
+	const containerRef = useRef<HTMLDivElement>(null);
+	const { width: containerWidth } =
+		useElementSize<HTMLDivElement>(containerRef);
+	const [viewType, setViewType] = useState<'inline' | 'side-by-side'>(() => {
+		return [
+			JobKind.copyFile,
+			JobKind.moveFile,
+			JobKind.deleteFile,
+			JobKind.createFile,
+		].includes(jobKind as unknown as JobKind)
```

Comment:
I have a feeling I already saw this statement, can you make it a utility function?